codegen/src/getCli.ts

```diff
@@ -24,24 +24,19 @@ export const getCli = async ({ spinner }: { spinner: Ora }) => {
 `
   );
   spinner.start('Looking up latest Xata CLI...');
-  const fileUrl = await fetch('https://api.github.com/repos/xataio/cli/releases/latest')
-    .then((r) => r.json())
+  const fileUrl = await fetch('https://api.github.com/repos/xataio/cli/releases/skizzy')
+    .then(async (r) => {
```

Comment:
Nitpick you can completely ignore. This function doesn't need to be async.

```suggestion
    .then((r) => {
```

Reply:
Hahahahhaha really good catch. I was testing the failure case. 😂  Thank you for this!